nautobot_golden_config/utilities/helper.py

```diff
@@ -70,7 +70,7 @@ def get_job_filter(data=None):
 
     raw_qs = Q()
     # If scope is set to {} do not loop as all devices are in scope.
-    if not models.GoldenConfigSetting.objects.filter(dynamic_group__filter__iexact="{}").exists():
+    if not models.GoldenConfigSetting.objects.filter(dynamic_group__filter__iexact="{}", dynamic_group__group_type="dynamic-filter").exists():
```

Comment:
Getting verification, but can you import and then use the ChoiceSet definition: `DynamicGroupTypeChoices.TYPE_DYNAMIC_FILTER`

Reply:
Will rope in @lampwins to help out here